docs/docs/configure/skills.md

```
@@ -28,7 +28,75 @@ Focus on the query: $ARGUMENTS
+```yaml
+---
+name: house-rules
+alwaysApply: true
+description: ...
+---
+```
+
+At session start, after the standard `<available_skills>` listing, every matched
+skill body is appended to the system prompt under:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)